### PR TITLE
Enable plateform combobox, including old session

### PR DESCRIPTION
**Pull request by SebastienPeillet**
@pblottiere

### CODE REVIEW

File: src/core/session.py [hunk 1]
```diff
@@ -208,6 +208,20 @@ def init(self, directory: str, load: bool = True) -> None:
...
+                self._followersLayer,
+                self._observersLayer,
+                self._sightingsLayer,
+                self._environmentLayer,
+                self._surveyLayer,
+                self._transectLayer,
+                self._strateLayer,
+                self._plateformLayer,
+            ]:
+                layer._init(layer.layer)
```
Comment (SebastienPeillet) [inline src/core/session.py:224]: NB : This is necessary to update old project

Comment (pblottiere) [reply src/core/session.py:224]: Ok :+1:.

We'll probably need an update mechanism like in stormm-ground at one point (but it's not necessary for now).

**Comment by pblottiere** [comment]
@SebastienPeillet thanks! +1 to merge and release :)

**Comment by pblottiere** [comment]
(btw, you probably can update the changelog in this PR)

**Pull request merged**